index.rst

```diff
@@ -240,6 +240,101 @@ fitSrcMatchAstrometry
+   Its output catalog, ``initial_source_detector``, will be converted to Parquet before it is written, but not SDM-standardized.
+   Footprints will be written to a separate ``SourceCatalog`` dataset, ``initial_footprints``.
+   Whether this task does multiple detection rounds (to iterate on CR detection, the source detection filter, or the detection threshold) is TBD; we'd like to minimize that.
+   We would like to avoid running a deblender here (if we do, it will have to happen after we have the PSF model), but this depends on how this performs on crowded fields.
+   We are not thrilled with the name of this task and would love ideas for improvements (which should be coordinated with the names for ``bootstrapVisit``, ``finalizeImage``, ``compressImage``, and ``rebuildImage``).
+
+bootstrapVisit
+   Consolidate the per-detector outputs of ``bootstrapImage`` and recover from failures on some detectors by using those that succeeded (especially for WCSs).
+
+associateIsolatedStars
+   Pretty much just a renamed ``IsolatedStarAssociationTask``.
+
+solveAstrometry
+   Pretty much just a generic label for ``jointcal`` and ``GBDES``.
+
+fgcm
+   Nothing new here, except the sharding and names of the output datasets.
+
+modelVisitBackground
+   A replacement for SkyCorrectionTask, probably using PCA.
+   SkyCorrectionTask will serve as a placeholder until we have something better.
+   Also takes care of subtracting the wings of bright stars.
+
+finalizeCharacterization
+   Nothing new here except new names for the output datasets.
+
+   We are using "characterizations" here to mean "PSFs and aperture corrections", and are not thrilled about that, but we need something that means "PSFs and aperture corrections" that could also absorb aperture corrections being done rather differently than they are today.
+
+finalizeAstrometry
+   This is a re-run of the astrometry solver with slightly different connections; I think it'll be necessary to make our (achromatic) WCSs consistent with the chromatic PSF models that we'll someday produce in ``finalizeCharacterization``.
+   But this needs more thought.
+
+updateVisitSummary
+   Nothing new here (though the current task is only a few weeks old, and apparently still has some bugs).
+
+finalizeImage
+   This new task takes all of our hard-won final characterizations and calibrations of the image and produces a final ``{visit, detector}`` ``pvi`` image and full-depth, all-measurements ``final_source_detector`` catalog.
+   The latter will be SDM-standardized before it is ever written to disk.
+
+consolidateSourceTable
+   This task just concatenates the ``final_source_detector`` catalogs into a single per-visit ``source`` catalog.
+
+compressImage
+   This task reads the ``pvi`` dataset, lossy-compresses the image and variance planes, lossless-compresses everything else (at least the mask; I don't know if compressing more than that is possible).
```

Comment:
I am really not sure why you want to do lossy-compression, there are some really impressive lossless compression algorithms. And on that note, what is the average percentage difference in storage space between lossy compressed images vs lossless?

Reply:
Robert Gruendl did an analysis at some point: https://dmtn-068.lsst.io

Without lossy compression the assertion was that we would not be able to archive the PVIs at all and would have to regenerate them on demand. It was a big struggle to include the PVIs in the costing model at all.

Reply:
Echoing Tim here: I don't think there are lossless compression of _floating point_ images that get anywhere near the compression ratios we need to make saving PVIs viable.  Lots of prior art here, but I don't know the details.